packages/ui/app/src/api-page/endpoints/EndpointContent.tsx

```diff
@@ -109,16 +139,23 @@ export const EndpointContent = React.memo<EndpointContent.Props>(function Endpoi
                     <div className="mt-8 flex">
                         <div className="flex flex-1 flex-col gap-12">
                             {endpoint.path.pathParameters.length > 0 && (
-                                <PathParametersSection pathParameters={endpoint.path.pathParameters} />
+                                <PathParametersSection
+                                    pathParameters={endpoint.path.pathParameters}
+                                    getParameterAnchor={(param) => computeAnchor("path", param)}
```

Comment:
no need to change but I think these callbacks would've been a good candidate for a context because they get passed around a ton and don't need to be modified (I think?) between here and when they get invoked